Dockerfile

```diff
@@ -1,47 +1,15 @@
-FROM python:3.8.6-alpine as base
+FROM python:3.8.16-slim
 
-####
+RUN apt update && \
```

Comment:
Could you specify some things smaller than libgtk-3-dev, because libgtk-3-dev will pull a lot of dependencies, while we only need gdk-pixbuf lib.